.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
```

Comment:
The `.DS_Store` file does't need to be included in the pull request. Can you remove it? You could also add the `.DS_Store` file to your .gitignore file so git always ignores it.